WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -114,8 +114,44 @@ extension XCUIElement {
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
 
-    public func getTextVisibilityCount(text: String) throws -> Int {
-        let predicate = NSPredicate(format: "label CONTAINS[c] %@", text)
+    func getStaticTextVisibilityCount(textToFind: String) throws -> Int {
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", textToFind)
         return XCUIApplication().staticTexts.containing(predicate).count
     }
+
+    public func assertTextVisibilityCount(textToFind: String) {
+        let count = try! getStaticTextVisibilityCount(textToFind: textToFind)
+        XCTAssertTrue(count == 1, "Expected '\(textToFind)' to appear once, but it appeared '\(count)' times!")
+    }
+
+    // Parent is using accessibilityIdentifier, child is using staticText
+    func verifyChildElementOnParentCell(parent: String, child: String) throws -> Bool {
+        let parentPredicate = NSPredicate(format: "identifier == %@", parent)
+        let childPredicate = NSPredicate(format: "label ==[c] %@", child)
+
+        return XCUIApplication().tables.cells.matching(parentPredicate).children(matching: .staticText).element(matching: childPredicate).firstMatch.exists
```

Comment:
Out of scope for this PR, but, IMHO, it's best to build these methods so that they don't instantiate `XCUIApplication` directly, but rather let the caller decide on which instance to run the assertions and interactions.

Something like this:

```swift
extension XCUIApplication {

  func exists(elementMatchingIdentifier parentIdentifier: String, containingChildWithLabel childLabel: String) throws -> Bool {
        let parentPredicate = NSPredicate(format: "identifier == %@", parentIdentifier)
        let childPredicate = NSPredicate(format: "label ==[c] %@", childLabel)

        return tables
            .cells
            .matching(parentPredicate)
            .children(matching: .staticText)
            .element(matching: childPredicate)
            .firstMatch
            .exists
    }
}
```

I'm suggesting this so that the code using this method can pass an already existing `XCUIApplication` instance, instead of creating a new one.

```swift
if app.exists(elementMatchingIdentifier: "foo", containingChildWithLabel: "bar") {
  // do something
} else {
  // do something different
}
```

In the context of `ScreenObject` subclasses, all instances already have access to an `app` property that must be passed at `init` time.

Reply:
Thanks for the suggestion and explanation, I'll make the update so that they don't instantiate `XCUIApplication` directly in the next PR.